packages/core/test-utils/src/utils.js

```diff
@@ -588,14 +589,41 @@ function prepareBrowserContext(
         return null;
       },
     },
-    currentScript: {
-      src: 'http://localhost/script.js',
-    },
+
+    currentScript: null,
   };
 
   var exports = {};
+
+  class customError {
```

Comment:
nit: UpperCamelCase? and maybe a better name

Also, I wonder if this should be a function with a `.prototype` property that's the current Error.prototype. Otherwise assumptions about this being the "real" Error may fail.